### ISSUE TITLE
Grid on the Kitchen Sink doesn't seem to be accessible with NVDA

### ISSUE THREAD

**Issue by frastlin** [zurb/foundation-sites#9716]
Hello,
I am new to Foundation, but I really love the commitment the community has to accessibility.
I was testing out all the components at:
http://foundation.zurb.com/sites/docs/kitchen-sink.html#0
and the Grid examples are weird. They seem to just be text within a div element.
Here is what I get with a copy and paste:

2/3/4 columns
4/3/4 columns
6/6/4 columns

When I see grid, I am expecting something like Google Sheets or:
https://www.w3.org/TR/wai-aria-practices/examples/grid/dataGrids.html#

If not that, at least a table.

I am specifically looking for a framework that implements the above grid and from what it looks like,the foundation grid should be that, but entering edit mode didn't work.
Please let me know if I am missing something!

**Comment by IamManchanda** [comment]
OMG!!

**Comment by frastlin** [comment]
After reading the Grid documentation, it appears as if the grid is only a layout tool. Apparently it is meant to be table styling and not a grid as Aria defines a grid.
I hope I'm wrong, but reading the HTML there are no role states defined. There could be the role defined in Javascript for the grid class, but this is unlikely as NVDA does not read column or row which it would if there were aria attributes defined in the code.
So I will take this as a naming confusion and close the issue.

**Comment by IamManchanda** [comment]
**YAII**
